styleguide/derek/solutions/solutions.ejs

```diff
@@ -17,9 +17,9 @@
 </div>
```

Comment:
Please update the link above to point to the new solutions-usage URL. 
https://github.com/rackerlabs/zoolander/pull/293/commits/c2f20da020cb28fa9661e9de89eb6a3a84b72428#diff-8a37516e22a27a627a72eb50bfff1809R5